torch/autograd/gradcheck.py

```diff
@@ -493,6 +547,14 @@ def get_notallclose_msg(analytical, numerical, output_idx, input_idx, error_str=
     return error_str + 'Jacobian mismatch for output %d with respect to input %d,\n' \
         'numerical:%s\nanalytical:%s\n' % (output_idx, input_idx, numerical, analytical)
 
+def transpose(m):
```

Comment:
nit: m -> matrix_of_tensors. I needed to think for a bit about what was actually getting transposed

Also, there's a one-liner for this (that is slightly different, returns a List[Tuple[Tensor]] if that matters)
```
def transpose(m):
    return zip(*m)
```

Reply:
So clean!